rubix-bookkeeper/src/main/java/com/qubole/rubix/bookkeeper/BookKeeper.java

```diff
@@ -444,6 +477,21 @@ public void onRemoval(final RemovalNotification<String, FileMetadata> notificati
         .build();
   }
 
+  private static void initializeFileInfoCache(final Configuration conf)
+  {
+    fileInfoCache = CacheBuilder.newBuilder()
+        .expireAfterWrite(36000, TimeUnit.SECONDS)
+        .removalListener(new RemovalListener<String, FileInfo>()
+        {
+          @Override
+          public void onRemoval(RemovalNotification<String, FileInfo> notification)
+          {
+            log.info("removed FileInfo for path " + notification.getKey() + " due to " + notification.getCause());
```

Comment:
Just want to make sure you mean to log every eviction at info level.

Reply:
Yes. Just logging when the FileInfo is expiring. We can move it to Debug level as well.